libs/langchain-community/scripts/create-entrypoints.js

```diff
@@ -56,6 +57,7 @@ const entrypoints = {
   "llms/cloudflare_workersai": "llms/cloudflare_workersai",
   "llms/cohere": "llms/cohere",
   "llms/fireworks": "llms/fireworks",
+  "llms/googlegenerativeai": "llms/googlegenerativeai",
```

Comment:
Add to `requiresOptionalDependency` below as well

Reply:
Done